src/view/components/common/login-component/login-component.ts

```diff
@@ -0,0 +1,201 @@
+import {Component, Input, Inject, EventEmitter, Output, ViewEncapsulation} from '@angular/core';
+import {MD_INPUT_DIRECTIVES} from '@angular2-material/input/input';
+import {MdButton} from '@angular2-material/button';
+import {MdToolbar} from '@angular2-material/toolbar';
+import {MdCheckbox} from '@angular2-material/checkbox/checkbox';
+import {MD_CARD_DIRECTIVES} from '@angular2-material/card';
+import {LoginService} from '../../../../api/services/login-service';
```

Comment:
I'm trying to setup in systemjs that we can do ```/api/services/``` instead of ```../../../../``` haven't found the correct way to do it.